conanfile.py

```diff
@@ -2,25 +2,24 @@
 
 class libiconvConan(ConanFile):
     name = "libiconv"
-    version = "1.15.0"
-    branch = "master"
+    lib_version = "1.15.0"
+    revision = "0"
+    version = '{}-{}'.format(lib_version, revision)
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    description = "Convert text to and from Unicode"
+    url = "https://github.com/Pix4D/libiconv"
+    license = "LGPL-3.0"
     generators = "cmake"
-    settings =  "os", "compiler", "arch", "build_type"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = "shared=False", "fPIC=True"
-    url = "http://github.com/kwallner/libiconv"
-    scm = { "type": "git", "url": "auto", "revision": "auto" }
-    no_copy_source = True
-    
-    def config_options(self):
-        del self.settings.compiler.libcxx
+    exports_sources = "*"
+    options = {"shared": [True, False]}
+    default_options = "shared=False"
```

Comment:
I would use a dictionary here, as this is not as prone to typos.

Reply:
done